app/domain/sac_imports/wso21_people_importer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacImports
+  class WSO21PeopleImporter
+    REPORT_HEADERS = [
+      :navision_id,
+      :first_name,
+      :last_name,
+      :errors
+    ]
+
+    def initialize(output: $stdout)
+      truemail_with_regex
+      @output = output
+      @source_file = CsvSource.new(:WSO21)
+      @csv_report = CsvReport.new(:"7_wso21_people", REPORT_HEADERS)
+    end
+
+    def create
+      data = @source_file.rows
+      data.each do |row|
+        process_row(row)
+      end
+      @csv_report.finalize(output: @output)
+    end
+
+    private
+
+    def basic_login_group
+      @basic_login_group ||= Group::AboBasicLogin.first!
+    end
+
+    def abo_group
+      @abo_group ||= Group::AboTourenportal.first!
+    end
```

Comment:
@mtnstar @openscript Kann ich das so machen? Diese Gruppen müssen ja immer vorhanden sein und sind quasi Singletons, richtig?

Reply:
ja, die beiden Gruppen sind aktuell unique und wird sich während der Zeit in der wie diese Scripts laufen lassen auch nicht ändern.